cdap-watchdog/src/main/java/co/cask/cdap/logging/write/FileMetaDataManager.java

```diff
@@ -272,7 +272,8 @@ public String call() throws Exception {
                 }
 
                 if (Strings.isNullOrEmpty(namespacedLogDir)) {
-                  LOG.warn("Delete metadata for {} namespace", namespaceId.getEntityName());
+                  LOG.warn("File {} is not present because namespace {} does not exist",
```

Comment:
I just realize that the log above should be warning not an error (line 263, but github does not allow me to comment there).